.github/workflows/actionlint.yml

```diff
@@ -0,0 +1,13 @@
+name: Lint GitHub Actions Workflows
+on:
+  push:
+  pull_request:
+permissions:
+  contents: read
```

Comment:
Is contents enough for the actionlint action? I only ask because of the packages permission that was needed for the signore container.

Reply:
I would presume this is enough since the action only runs a command and doesn't modify the contents of the repo

Reply:
While fixing the go-test.yml action, I did trigger this one, and I can confirm that with this permission alone, it works!